crates/shared/src/price_estimation/mod.rs

```diff
@@ -245,6 +246,39 @@ pub struct Arguments {
 
     #[clap(flatten)]
     pub balance_overrides: balance_overrides::Arguments,
+
+    /// List of mappings of native price tokens substitutions with approximated
+    /// value from other token:
+    /// "<token1>|<approx_token1>,<token2>|<approx_token2>"
+    /// - token1 is a token address for which we get the native token price
+    /// - approx_token1 is a token address used for the price approximation
+    ///
+    /// It is very important that both tokens in the pair have the same number
+    /// of decimals.
+    #[clap(
+        long,
+        env,
+        value_delimiter = ',',
```

Comment:
```suggestion
        value_delimiter = true,
```
I think that should be enough. We already have a similar construction here:
https://github.com/cowprotocol/services/blob/main/crates/autopilot/src/arguments.rs#L150-L153

Reply:
In the linked code from the `autopilot` we are using `use_value_delimiter = true`, which is deprecated from clap v4.0.0 ([link](https://docs.rs/clap_builder/4.5.29/src/clap_builder/builder/arg.rs.html#1644)), so I would prefer to stick to `value_delimiter = ','`.